learning/tour-of-beam/frontend/lib/pages/tour/widgets/group_title.dart

```diff
@@ -75,11 +78,14 @@ class _GroupProgressIndicator extends StatelessWidget {
         }
 
         return Container(
-          margin: const EdgeInsets.symmetric(horizontal: BeamSizes.size6),
-          height: BeamSizes.size8,
-          width: BeamSizes.size8,
+          margin: const EdgeInsets.only(
+            left: BeamSizes.size7,
+            right: BeamSizes.size10,
+          ),
+          height: _diameter,
+          width: _diameter,
           child: CircularProgressIndicator(
-            strokeWidth: BeamSizes.size3,
+            strokeWidth: _thickness,
             color: BeamColors.green,
             backgroundColor: Theme.of(context)
                 .extension<BeamThemeExtension>()!
```

Comment:
Looks like just another case for `CompletenessIndicator`.

Reply:
`CompletenessIndicator` is binary with either an empty circle or check. `CircularProgressIndicator` and `progress` are not used in unit progress indication.

Reply:
Made a separate boolean and double indicators.